src/plugin.json

```diff
@@ -34,8 +34,7 @@
     "updated": "%TODAY%"
   },
   "dependencies": {
-    "grafanaVersion": "7.5.0",
```

Comment:
grafanaVersion is deprecated (https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#properties-1)